jetbrains-core/resources/META-INF/plugin.xml

```diff
@@ -222,15 +222,18 @@
                     class="software.aws.toolkits.jetbrains.services.cloudformation.DeleteCloudFormationStackAction"/>
         </group>
 
+        <group id="aws.toolkit.explorer.s3" popup="false" compact="false">
```

Comment:
I think it is better to use the original group id `aws.toolkit.explorer.s3.bucket`.

Reply:
@kiiadi suggested that create bucket should be attached to the S3 service node.

Reply:
Ah make sense.